pkg/utils/utils.go

```diff
@@ -15,6 +15,8 @@ var (
 	NetDirectory = "/sys/class/net"
 	// SysBusPci is sysfs pci device directory
 	SysBusPci = "/sys/bus/pci/devices"
+	// UserspaceDrivers is a list of driver names that don't have netlink representation for their devices
+	UserspaceDrivers = []string{"vfio-pci", "uio_pci_generic", "igb_uio"}
```

Comment:
we don't support uio_pci_generic, igb_uio it intel drivers.

Reply:
I will remove these drivers

Reply:
I dont think intel supports this ATM, so im ok with only keeping vfio-pci for now. later on it can be extended to other user-space drivers if needed